Meadowlark-js/packages/meadowlark-core/src/handler/CrudHandler.ts

```diff
@@ -107,7 +98,7 @@ export async function create(event: APIGatewayProxyEvent, context: Context): Pro
     }
 
     const resourceId = documentIdForEntityInfo(entityInfo);
-    const { result, failureMessage } = await createEntity(
+    const { result, failureMessage } = await backendPlugin().createEntity(
```

Comment:
I had to stop and think about this and the dependency injection pattern. Seems like a variant on the Service Lookup pattern.